openrr-nav-viewer/src/parser.rs

```diff
@@ -57,3 +58,24 @@ pub fn parse_robot_pose_to_point(robot_pose: &Pose, color: Color32, point_radius
 
     Points::new(plot_point).color(color).radius(point_radius)
 }
+
+pub fn parse_robot_pose_to_polygon(robot_pose: &Pose, color: Color32, size: f64) -> Polygon {
+    let size = size * 0.04;
```

Comment:
I feel this is ambiguous.
If you are converting real size into canvas size or something, how about adding the word as the variable name?